### PR TITLE
Use data folder for showcase

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

**Comment by MattiSG** [comment]
Blocked by https://github.com/openfisca/openfisca.org/issues/98.

File: layouts/index.html
Comment (MattiSG) [inline layouts/index.html:70]: `/img/revenu-de-base.jpg` seems to have been duplicated into `/static/img/showcase/revenu-de-base.jpg` instead of being moved 😉

File: layouts/index.html
Comment (MattiSG) [inline layouts/index.html:37]: `/img/mes-aides.png` seems to have been duplicated instead of being moved 😉

**Review by MattiSG — CHANGES_REQUESTED** [review]
It's hard not to correct the content since all of this is clearly obsolete, but let's focus on the data format 👍

File: layouts/index.html
Comment (clementbiron) [reply layouts/index.html:33]: I don't really understand your comment, it would be easier for me to discuss it.

As I remember, we had discussed not using a categories because reuses often belong to several categories, instead we have prototyped  the using of tags, which makes me see that I have totally forgotten them in this PR. Do you want me to implement this metadata `tags` ? 

If yes i can suggest two different solutions where `tags` reuses metadata 
-  contains the values in French and English (wordy but easy for the contribution)

```yml
tags:
  fr: [sondage, python, gouvernement]
  en: [survey, python, governement]
```

- contains `tag` ID (allows better control of the list of tags but it is less easy to contribute because you have to refer to a data file that lists the available tags and related identifiers)

```yml
tags: [2, 3, 7]
```
```yml
tags:
  - id: 2
    fr: gouvernement
    en: governement
  - id: 3
    fr: sondage
    en: survey
  - id: 7
    fr: python
    en: python
```

File: layouts/index.html
Comment (clementbiron) [reply layouts/index.html:70]: Sorry about that.
I'll have to rewrite the commit before merge, right?

Comment (MattiSG) [reply layouts/index.html:70]: Yes, that would save storage.

File: layouts/index.html
Comment (MattiSG) [reply layouts/index.html:33]: No, you are right. This is an optimisation for the current layout but irrelevant for the future. Forget what I said 🙂 Sorry for the noise!

**Comment by MattiSG** [comment]
I just rebased on top of #101 to fix the build, and fixed the duplicate images issues along the way. I also optimised the images. Please remember to optimise assets before committing them to minimise storage usage 🙂 🌱

**Review by MattiSG — APPROVED** [review]
🧑‍🎨 🚀

**Pull request merged**
